orangecontrib/spectroscopy/widgets/owfft.py

```diff
@@ -580,8 +609,15 @@ def check_metadata(self):
             self.controls.phase_corr.setDisabled(True)
             self.controls.phase_res_limit.setDisabled(True)
             self.controls.phase_resolution.setDisabled(True)
+            self.controls.wavenumber_scaling_factor.setDisabled(False)
+            self.controls.rescale_wavenumber.setDisabled(False)
 
             info = self.data.attributes
+            try:
```

Comment:
Additions to this widget should also be tested (make a test, that inputs some data without the scaling factor, check that it remains 1, add the scaling to `data.attributes`, input again, and confirm that it is set.

Reply:
Thanks @markotoplak , I missed from the title this PR was also changing FFT. I would prefer to keep them separate in the future.

What is the physical origin of this scaling factor?